docs/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_fluid_loop_attached_to_boiler.md

```diff
@@ -13,10 +13,10 @@
 
 ## Logic:   
 - Set is_hvac_sys_fluid_loop_attached_to_boiler equal to FALSE: `is_hvac_sys_fluid_loop_attached_to_boiler = FALSE`  
-- For each boiler in B_RMR, save boiler to loop boiler dictionary: `for boiler_b in B_RMR.RulesetModelInstance.boilers: loop_boiler_dict[boiler_b.loop].append(boiler_b)`
-- Create an object associate with the heating_system fluid loop associated with hvac_b: `fluid_loop_b = hvac_b.heating_system[0].hot_water_loop`
+- For each boiler in B_RMR, save boiler to loop boiler list: `for boiler_b in B_RMR.RulesetModelInstance.boilers: loop_boiler_id.append(boiler_b.loop)`
```

Comment:
Should `loop_boiler_id` be plural to indicate it is a list of ids?
Would `boiler_loop_ids` be even more descriptive?